modules/scheduler/gke-cluster/main.tf

```diff
@@ -30,6 +30,12 @@ locals {
   }]
 
   sa_email = var.service_account_email != null ? var.service_account_email : data.google_compute_default_service_account.default_sa.email
+
+  # additional VPCs enable multi networking 
+  derived_enable_multi_networking = length(var.additional_networks) > 0 ? true : var.enable_multi_networking
```

Comment:
I think you can avoid this by having multinetwork output the `enable_multi_networking: true` field

Reply:
> I think you can avoid this by having multinetwork output the `enable_multi_networking: true` field

I'm not exactly sure what this means. Does it mean to add an output "enable_multi_networking" to multinetwork module? I wonder isn't multivpc a generic module and enable_multi_networking a gke-only setting?

Reply:
👍🏻 if similar data are needed in different formats, it's fine to have multiple output values.

Reply:
@nick-stroud while you are reviewing to see if I maybe solved this another way, I added enable_multi_networking output to multivpc module. It won't affect my changes on gke-cluster logic.

Reply:
enable_multi_networking output removed from multivpc module.